server/src/api/commands.ts

```diff
@@ -1,49 +1,117 @@
 import * as os from 'os';
 
-const spawn = require('node-pty-prebuilt').spawn;
-
-interface CommandResult {
+interface CommandInformation {
+  id: string;
+  workspace: string;
```

Comment:
This is the full workspace path correct? (Might be good to change the name to workspacePath to differentiate between workspace names)

Reply:
this is a workspace name (from package.json), we can replace it with 'path' if it helps